CExcercises/Ex3-genericAdder.c

```diff
@@ -57,15 +57,17 @@ int main (int argc, char **argv)
 
 int genericPlus(const void* array, size_t elementSize, size_t fieldOffset, size_t amountOfElements)
 {
-	int i = 0;
+	// Like te return tyoe is int, assume that all the fields to add are int
+	int indexOfStructs = 0;
 	int sumResult = 0;
-	struct S * sArray  = (struct S *) array; // Do with pointer
+	struct S * sArray  = (struct S *) array; // Cast to Struct S
 
-	for(i=0; i < amountOfElements; i = i + 1)
+	for(indexOfStructs=0; indexOfStructs < amountOfElements; indexOfStructs ++)
```

Comment:
But, it's the same result.

http://stackoverflow.com/questions/4706199/post-increment-and-pre-increment-within-a-for-loop-produce-same-output